tests/patch-series.test.ts

```diff
@@ -405,6 +405,36 @@ Fetch-It-Via: git fetch ${repoUrl} my-series-v1
             expect(log).toHaveBeenCalledTimes(1); // verify no more errors
         });
 
+        test("CC footer parsing", async () => {
+            const prTitle = "My test PR!";
+            const prBody = [
+                "some description goes here",
+                "",
+                "Cc: Some Contributor <contributor@example.com>",
+                "Cc: Git Maintainer <maintainer@gmail.com>",
+            ].join("\r\n");
+
+            const senderName = "Current Patch Author";
+            const senderEmail = "author@somewhere.org";
+            const parsed = await PatchSeries.parsePullRequest(".", prTitle,
+                                                              prBody,
+                                                              senderName,
+                                                              senderEmail);
```

Comment:
Passing `'.'` is passing the gitgitgadget repo when a git repo is expected.  Try/catch will handle but could be interesting if gitgitgadget/gitgitgadget adds a pull request template that says `some description goes here`.  🙂

Reply:
Thanks for pointing out!

I also noticed that something is wrong https://github.com/gitgitgadget/gitgitgadget/pull/163#issuecomment-564218304 but then had to switch to other task.

Fixed now.